db/migrate/20151110082345_add_metadata_to_spotlight_resource.rb

```diff
@@ -1,5 +1,12 @@
 class AddMetadataToSpotlightResource < ActiveRecord::Migration
   def up
-    add_column :spotlight_resources, :metadata, :blob
+    # postgresql does not have a blob type
+    conn_type = connection.adapter_name.downcase.to_sym
+    field_type = (conn_type == :postgresql) ? :bytea : :blob
+    add_column :spotlight_resources, :metadata, field_type
```

Comment:
Instead of having adapter specific type definitions, I wonder if we could simply use `:binary` in all cases. I believe this is the ActiveRecord native type that is intended to map to `:bytea` for postgres and `:blob` for MySQL/sqlite.